src/components/Counter.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Button from "./button/Button";
+import Title from "./title/Title";
+export default class Counter extends React.Component {
+    constructor({initialValue}) {
+        super();
+        this.state={
+            initialValue : initialValue
+        }
+    }
+    counterUpdate (priority) {
+        this.setState({initialValue: priority ? this.state.initialValue + 1 : this.state.initialValue - 1 })
```

Comment:
a kind of anti pattern, better use setState((prevState) => {prevState + 1});

Reply:
Done